R/target_sda.R

```diff
@@ -263,6 +263,7 @@ compute_ald_adjusted_scenario <- function(data, corporate_economy) {
     ungroup()
 
   data %>%
+    filter(.data$year >= min(corporate_economy$year)) %>%
```

Comment:
Just checking, we don't need `na.rm = TRUE` in this call to `min()`?

Reply:
I think it wouldn't hurt to add